docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,351 @@
+Here is configuration file that does not add or remove any Container capabilities:
+
+{% include code.html language="yaml" file="security-context-3.yaml" ghlink="/docs/tasks/configure-pod-container/security-context-3.yaml" %}
+
+Create the Pod:
+
+```shell
+kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/security-context-3.yaml
+```
+
+Verify that the Pod's Container is running:
+
+```shell
+kubectl get pod security-context-demo-3
+```
+
+Get a shell into the running Container:
+
+```shell
+kubectl exec -it security-context-demo-3 -- sh
+```
+
+In your shell, list the running processes:
+
+```shell
+ps aux
+```
+
+The output shows the process IDs (PIDs) for the Container:
+
+```shell
+USER  PID %CPU %MEM    VSZ   RSS TTY   STAT START   TIME COMMAND
+root    1  0.0  0.0   4336   796 ?     Ss   18:17   0:00 /bin/sh -c node server.js
+root    5  0.1  0.5 772124 22700 ?     Sl   18:17   0:00 node server.js
+```
+
+In your shell, view the status for process 1:
+
+```shell
+cd /proc/1
+cat status
+```
+
+The output shows the capabilities bitmap for the process:
+
+```
+...
+CapPrm:	00000000a80425fb
+CapEff:	00000000a80425fb
+...
+```
+
+Make a note of the capabilities bitmap, and then exit your shell:
+
+```shell
+exit
+```
+
+Next, run a Container that is the same as the preceding container, except
+that it has additional capabilities set.
+
+Here is the configuration file for a Pod that runs one Container. The configuration
+adds the `CAP_NET_ADMIN` and `CAP_SYS_TIME` capabilities:
+
+{% include code.html language="yaml" file="security-context-4.yaml" ghlink="/docs/tasks/configure-pod-container/security-context-4.yaml" %}
+
+Create the Pod:
+
+```shell
+kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/security-context-4.yaml
+```
+
+Get a shell into the running Container:
+
+```shell
+kubectl exec -it security-context-demo-4 -- sh
+```
+
+In your shell, view the capabilities for process 1:
+
+```shell
+cd /proc/1
+cat status
+```
+
+The output shows capabilities bitmap for the process:
+
+```shell
+...
+CapPrm:	00000000aa0435fb
+CapEff:	00000000aa0435fb
+...
+```
+
+Compare the capabilities of the two Containers:
+
+```
+00000000a80425fb
+00000000aa0435fb 
+```
+
+In the capability bitmap of the first container, bits 12 and 25 are clear. In the second container,
+bits 12 and 25 are set. Bit 12 is `CAP_NET_ADMIN`, and bit 25 is `CAP_SYS_TIME`.
+See [capability.h](https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h)
+for definitions of the capability constants.
+
+**Note**: Linux capability constants have the form `CAP_XXX`. But when you list capabilities
+in your Container manifest, you must omit the `CAP_` portion of the constant. For example,
+to add `CAP_SYS_TIME`, include `SYS_TIME` in your list of capabilities.
+
+## Assign SELinux labels to a Container
+
+To assign SELinux labels to a Container, include the `seLinuxOptions` field in
```

Comment:
Note: The host OS must have the SELinux security module loaded in order for SELinux labels to have any affect. If the module is not loaded, I'm not sure whether setting them will give an error, or silently do nothing.